zorkington-game.js

```diff
@@ -0,0 +1,503 @@
+class House {
+    constructor() {
+        this.roomCollection = [];
+    }
+
+    addRoom(room) {
+        this.roomCollection.push(room);
+    }
+
+    // helper function to get room based on column and row
+    getRoom(col, row) {
+        let currRoom = null;
+
+        this.roomCollection.forEach((room) => {
+            if (col === room.column &&
+                row === room.row) {
+                currRoom = room;
+            }
+        })
+        return currRoom;
+    }
```

Comment:
This is a really unique concept for building this game and I like it. There's a minor big with it but that can be fixed with some logic, but the bug is essentially that even if there isn't a specified doorway the user can still make that move.